articles/monitoring-and-diagnostics/monitoring-overview-alerts-classic.md

```diff
@@ -24,7 +24,7 @@ In the past Azure Monitor, Application Insights, Log Analytics and Service Healt
 
 You can view classic alerts only in the classic alerts user screen in the Azure Portal. You get this screen from the **View classic alerts** button on the alerts screen. 
 
- ![Alert choices in Azure portal](./media/monitoring-overview-alerts-classic/monitor-alert-screen2.png)�
+ ![Alert choices in Azure portal](./media/monitoring-overview-alerts-classic/monitor-alert-screen2.png)
```

Comment:
There was a hidden character at the end of this that was changed in the first commit so I removed it and rebased